### PR TITLE
Gelu matcher fix

### PR DESCRIPTION
**Pull request by kahmed10**

### CODE REVIEW

File: src/include/migraphx/matcher.hpp [hunk 1]
```diff
@@ -567,6 +567,25 @@ inline auto has_value(T x, float tolerance = 1e-6)
     });
 }
 
+template <class T>
+inline auto has_value_multi(T x, float tolerance = 1e-6)
```
Comment (pfultz2) [inline src/include/migraphx/matcher.hpp:571]: Is a new matcher necessary? Can't we just rename this to `has_value`?

File: src/targets/gpu/fuse_ops.cpp [hunk 2]
```diff
@@ -335,14 +335,18 @@ void move_standard_front(std::vector<instruction_ref>& args)
 struct find_gelu
 {
 
+    static auto sqrt_1_2_val()
+    {
+        return match::any_of(match::has_value(M_SQRT1_2), match::has_value_multi(M_SQRT1_2));
```
Comment (pfultz2) [inline src/targets/gpu/fuse_ops.cpp:340]: If `has_value_multi` matches then `has_value` would match.

File: src/include/migraphx/matcher.hpp (hunk 1 above)
Comment (kahmed10) [reply src/include/migraphx/matcher.hpp:571]: I believe so. Initially I checked bert_mrpc, and the way we had written `has_value` was limited to literals of size 1. In the onnx file, it seems they had already broadcasted those values, so the matcher was incorrect.

File: src/targets/gpu/fuse_ops.cpp (hunk 2 above)
Comment (kahmed10) [reply src/targets/gpu/fuse_ops.cpp]: fixed

**Pull request merged**